src/development/tools/devtools/release-notes/release-notes-2.18.0-src.md

```diff
@@ -0,0 +1,88 @@
+# DevTools 2.18.0 release notes
+
+Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
+
+# Inspector updates
+
+- Auto scrolling behavior improved when snapping to a widget into focus -
+  [#4283](https://github.com/flutter/devtools/pull/4283)
+- Fix issue where widget inspector wouldn't load when connecting to a paused
+  app - [#4527](https://github.com/flutter/devtools/pull/4527)
+- Improve widget inspector hover cards to show progress while waiting for data -
+  [#4488](https://github.com/flutter/devtools/pull/4488)
+
+# Performance updates
+
+- Fix issue where scrollbar would go out of sync with the frame content -
+  [#4503](https://github.com/flutter/devtools/pull/4503)
+- Add offline support for raster stats -
+  [#4491](https://github.com/flutter/devtools/pull/4491)
+- Add 'Rendering time' column to Raster Metrics tab -
+  [#4474](https://github.com/flutter/devtools/pull/4474)
+
+# CPU profiler updates
+
+- Fix crash when an empty frame is filtered -
+  [#4502](https://github.com/flutter/devtools/pull/4502)
+- Fix bugs in CPU profile trees -
+  [#4413](https://github.com/flutter/devtools/pull/4413)
+- UI Cleanup - [#4404](https://github.com/flutter/devtools/pull/4404)
+
+# Memory updates
+
+- UI cleanup - [#4520](https://github.com/flutter/devtools/pull/4520)
+- Allocation tracing on multiple isolates is now supported -
+  [#4538](https://github.com/flutter/devtools/pull/4538)
+- Clear button added to allocation tracing tab -
+  [#4534](https://github.com/flutter/devtools/pull/4534)
+- Fix sorting column not being correctly updated -
+  [#4532](https://github.com/flutter/devtools/pull/4532)
```

Comment:
If we are advertising the new Allocation Profile and Tracing Tabs, we probably don't need to include release notes entries for polish items that were added as part of getting these two tabs ready. Think of it like this: the changelog is to list every commit that is included with our release, the release notes are to notify the user of user-facing features & fixes that are included with this release. So for example, if the user hasn't yet seen the allocation tracing tab, then they probably don't need to read the improvements we made to it before turning it on, if that makes sense.